src/managers/summary-manager.instrumentation.md

```
@@ -0,0 +1,86 @@
+
+| Function | Status | Spans |
+|----------|--------|-------|
+| readDayEntries | instrumented | 1 |
+| formatDailySummary | instrumented | 0 |
+| saveDailySummary | instrumented | 1 |
+| generateAndSaveDailySummary | instrumented | 1 |
+| getWeekBoundaries | instrumented | 0 |
+| readWeekDailySummaries | instrumented | 1 |
+| formatWeeklySummary | instrumented | 0 |
+| saveWeeklySummary | instrumented | 1 |
+| generateAndSaveWeeklySummary | skipped — LLM response had null parsed_output — no structured output was returned.
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)